remotespark/utils/sparkevents.py

```diff
@@ -0,0 +1,32 @@
+import remotespark.utils.configuration as conf
+
+
+class SparkEvents:
+    def __init__(self):
+        """
+        Create an instance from the handler mentioned in the config file.
+        """
+        events_handler = self._import_handler(conf.events_config()["handler"])
+        self.handler = events_handler()
+
+    def _import_handler(self, name):
+        """
+        Importing the handler based on the passed parameter.
+        """
+        components = name.split('.')
+        mod = __import__(components[0])
+        for comp in components[1:]:
+            mod = getattr(mod, comp)
+        return mod
```

Comment:
We decided to leave this decision for me, remember? And since it's going be a very tiny number of code lines. I found it's better to encapsulate it in a method.